models/issues/milestone.go

```diff
@@ -325,6 +379,15 @@ func DeleteMilestoneByRepoID(repoID, id int64) error {
 	if _, err = db.Exec(ctx, "UPDATE `issue` SET milestone_id = 0 WHERE milestone_id = ?", m.ID); err != nil {
 		return err
 	}
+
+	if err = m.LoadLabels(ctx); err != nil {
```

Comment:
Maybe just exec a SQL to delete by milestone_id is better.

Reply:
@lunny how it should look like? There seems to be no db.Exec SQL queries that delete in this project